app/javascript/src/components/ItemList/ItemListItem.jsx

```diff
@@ -17,7 +17,10 @@ export const ItemListItem = ({ item, children }) => {
   return (
     <div className="item-wrapper">
       <a className="item" href={adaptedItem.path}>
-        <div className="item-title">{adaptedItem.title}</div>
+        <div
```

Comment:
For the ones of us who don't do much React/Preact, is there a reason why this wouldn't work?

```js
<div className="item-title">filterXSS(adaptedItem.title)</div>
```

Reply:
This one of Gotchas of JSX https://shripadk.github.io/react/docs/jsx-gotchas.html

Reply:
Nice, thanks!